xwiki-platform-core/xwiki-platform-blog/xwiki-platform-blog-ui/src/main/resources/Blog/BlogParameters.xml

```diff
@@ -74,6 +73,6 @@
  * @param $imgName The name of the icon to use (without the extension).
```

Comment:
Would be nice to mention which name it corresponds to. In the past it was a silk icon name but now it's a generic icon set name, independent of the icon set used, right?